### ISSUE TITLE
Does not install with OCaml 4.06.0

### ISSUE THREAD

**Issue by Chimrod** [avsm/ezxmlm#5]
Due to the separation between Bytes and String, the package does not install with the latest version for OCaml.

The error does not come from the library, but the setup.ml script, and require only minor changes :

```bash
$ diff setup{,2}.ml
245c245
<     let buf = String.make (String.length s) 'X' in
---
>     let buf = Bytes.make (String.length s) 'X' in
366c366
<             "_"^buf
---
>             "_"^ (Bytes.to_string buf)
368c368
<             buf
---
>             (Bytes.to_string buf)
```

**Comment by shayne-fletcher** [comment]
I'm affected by this too. Any chance of a fix?

**Comment by Leonidas-from-XIV** [comment]
I think this can now be closed, I just built it fine on OCaml 4.07.1.

**Comment by avsm** [comment]
Confirmed! Sorry I missed this issue when it was posted
